lang/rust/README.md

```diff
@@ -526,7 +526,7 @@ fn main() -> Result<(), Error> {
     let mut record = Record::new(writer.schema()).unwrap();
     record.put("decimal_fixed", Decimal::from(9936.to_bigint().unwrap().to_signed_bytes_be()));
     record.put("decimal_var", Decimal::from((-32442.to_bigint().unwrap()).to_signed_bytes_be()));
-    record.put("uuid", uuid::Uuid::new_v4());
+    record.put("uuid", uuid::Uuid::parse_str("550e8400-e29b-41d4-a716-446655440000").unwrap());
```

Comment:
The idea of the example here is to _generate_ a new/fresh uuid

Reply:
Thanks for such quick response.

Reply:
Thanks for such a quick response. 

I returned it back.